Drv/Ip/IpSocket.cpp

```diff
@@ -219,8 +219,13 @@ SocketIpStatus IpSocket::recv(U8* data, U32& req_read) {
     for (U32 i = 0; (i < SOCKET_MAX_ITERATIONS) && (size <= 0); i++) {
         // Attempt to recv out data
         size = this->recvProtocol(data, req_read);
+
+        if ((size == -1) && (errno == EAGAIN)) {
```

Comment:
@JohanBertrand @csmith608 this file's changes should be kept, as they improve error handling (
`EAGAIN` is differentiated from `EINTR`).